server/optimus.go

```diff
@@ -190,10 +192,10 @@ func applicationKeyFromString(appKey string) (*[keyLength]byte, error) {
 }
 
 func (s *OptimusServer) setupDB() error {
-	// err := postgres.Migrate(s.conf.Serve.DB.DSN)
```

Comment:
@deryrahman any specific reason why this flow was disabled

Reply:
sorry, this is for debugging purpose in local, can you rebase? it should be removed